siddhi-stream-processor-archetype/src/main/resources/archetype-resources/src/main/java/__siddhi_stream_processor_name__StreamProcessor.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package ${package};
+
+import org.wso2.siddhi.core.config.ExecutionPlanContext;
+import org.wso2.siddhi.core.event.ComplexEvent;
+import org.wso2.siddhi.core.event.ComplexEventChunk;
+import org.wso2.siddhi.core.event.stream.StreamEvent;
+import org.wso2.siddhi.core.event.stream.StreamEventCloner;
+import org.wso2.siddhi.core.event.stream.populater.ComplexEventPopulater;
+import org.wso2.siddhi.core.exception.ExecutionPlanCreationException;
+import org.wso2.siddhi.core.executor.ConstantExpressionExecutor;
+import org.wso2.siddhi.core.executor.ExpressionExecutor;
+import org.wso2.siddhi.core.query.processor.Processor;
+import org.wso2.siddhi.core.query.processor.stream.StreamProcessor;
+import org.wso2.siddhi.query.api.definition.AbstractDefinition;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.extension.timeseries.linreg.RegressionCalculator;
+import org.wso2.siddhi.extension.timeseries.linreg.SimpleLinearRegressionCalculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * ${siddhi_stream_processor_name}
+ */
+
+public class ${siddhi_stream_processor_name}StreamProcessor extends StreamProcessor {
+
```

Comment:
@hmrajas , In the review we have asked you to write a specific scenario for streamProcessor Isn't ? Can you implement the stream processor  logic as similar to Time Window and include the event expiry time as additional attribute please ? That will be very clear..